### ISSUE TITLE
Resolve accessibility issues

### ISSUE THREAD

**Issue by wesley-dean-gsa** [GSA-TTS/tts.gsa.gov#106]
@katelandisgsa and crew ran WAVE and we now have results from Pa11y (PR #105) and some accessibility issues have been identified:

- [ ] The homepage image has the alt as "undefined". This is not a clearly defined alt. The icons in the footer have a similarly "undefined" alt text
- [ ] Use null alt text and no title attribute on img elements for images that AT should ignore. The purpose of this technique is to show how images can be marked so that they can be ignored by Assistive Technology.
- [ ] If no title attribute is used, and the alt text is set to null (i.e. alt="") it indicates to assistive technology that the image can be safely ignored. Note that having a "null" alt attribute is not the same as having no alt attribute.
- [ ] several many instances of `duplicate id attribute "svg-accessibility_new" found`

```
Error: Duplicate id attribute value "svg-accessibility_new" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #svg-accessibility_new
   └── <symbol viewBox="0 0 24 24" id="svg-accessibility_new" xmlns="http://www.w3.org/2000/svg"><path d="M20.5 6c-2.61.7-5.67 1...</symbol>
```
